### ISSUE TITLE
Error message with typescript project

### ISSUE THREAD

**Issue by viniciusvelasco** [azouaoui-med/react-pro-sidebar#3]
Hello, I'm using your great menu in a project with typescript template. I have the code below and it shows me the error Type '{children: string; } 'has no properties in common with type' IntrinsicAttributes & Props & RefAttributes <unknown> 'can help me how to solve? I don't believe it is in the project, but in some rule that typescript uses. Thank you

```<ProSidebar>
        <SidebarHeader></SidebarHeader>
        <SidebarContent>
          <Menu iconShape="circle">
            <MenuItem>Primeiro Menu</MenuItem>
          </Menu>
        </SidebarContent>
        <SidebarFooter></SidebarFooter>
      </ProSidebar>`
```

**Comment by azouaoui-med** [comment]
Fixed in [v0.4.1](https://github.com/azouaoui-med/react-pro-sidebar/releases/tag/v0.4.1)

**Comment by azouaoui-med** [comment]
Fixed in [v0.4.1](https://github.com/azouaoui-med/react-pro-sidebar/releases/tag/v0.4.1)
